### PR TITLE
Fix CVEs in golang.org/x/{net/sys} packages

### PR DESCRIPTION
**Pull request by humblec**
CVE-2021-44716
CVE-2022-29526

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

/kind cleanup

https://github.com/kubernetes-csi/livenessprobe/pull/153

```release-note
module dependencies are updated to latest version which also include CSI Spec version bump to v1.6 
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,13 +3,21 @@ module github.com/kubernetes-csi/livenessprobe
...
+	github.com/prometheus/client_golang v1.12.2 // indirect
+	github.com/prometheus/common v0.37.0 // indirect
+	github.com/prometheus/procfs v0.8.0 // indirect
+	golang.org/x/net v0.0.0-20220805013720-a33c5aa5df48 // indirect
+	golang.org/x/sys v0.0.0-20220804214406-8e32c043e418 // indirect
+	google.golang.org/genproto v0.0.0-20220804142021-4e6b2dfa6612 // indirect
+	google.golang.org/grpc v1.48.0
+	google.golang.org/protobuf v1.28.1 // indirect
+	k8s.io/component-base v0.24.3 // indirect
+	k8s.io/klog/v2 v2.70.1
```
Comment (pohly) [inline go.mod:20]: So this is mostly just a dependency update, i.e. business as usual. This part looks fine.

But why does go.mod have replace statements for golang.org/x/text? Why was that added and is that still relevant?

Comment (humblec) [reply go.mod:18]: Actually its not needed, I have reran it again which have removed that mention, ptal.. thanks @pohly

Comment (humblec) [reply go.mod:18]: That line was intact :) , anyway I have removed that as well @pohly ,

**Review by pohly — APPROVED** [review]
/lgtm

**Comment by humblec** [comment]
@xing-yang can you please approve this ? as this block the release tool update PR ?

**Comment by humblec** [comment]
@xing-yang @jingxu97 ptal.. thanks

**Comment by humblec** [comment]
@pohly may be you can approve this PR ?

**Comment by humblec** [comment]
As this is blocking release tool update PR 
/assign @jsafrane

**Pull request merged**